contracts/core/fee-calculators/PerformanceFeeCalculator.sol

```diff
@@ -208,6 +213,34 @@ contract PerformanceFeeCalculator is IFeeCalculator {
         return streamingFee.add(profitFee);
     }
 
+    /*
+     * Validate then set new streaming fee.
+     *
+     * @param  _newFeePercentage       Fee type and new streaming fee encoded in bytes
+     */
+    function adjustFee(
```

Comment:
I imagine an event should be emitted here?

Reply:
It is on the FeeCalculator